frontend/src/routes/Applications/Overview.tsx

```diff
@@ -1068,13 +1068,10 @@ export default function ApplicationsOverview() {
         emptyState={
           <AcmEmptyState
             key="appOverviewEmptyState"
-            title={t("You don't have any applications")}
+            title={t("You don't have any applications yet")}
             message={
               <Text>
-                <Trans
-                  i18nKey="Click <bold>Create application</bold> to create your resource."
-                  components={{ bold: <strong /> }}
-                />
+                <Trans i18nKey="To get started, create an application." />
```

Comment:
Do we need to keep the Trans component if we are not doing any formatting?

Reply:
Ah, I missed that. I'll take care of it.